travis-ymls/case-sensitive-paths-webpack-plugin.travis.yml

```diff
@@ -0,0 +1,39 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : case-sensitive-paths-webpack-plugin
+# Source Repo         : https://github.com/Urthen/case-sensitive-paths-webpack-plugin
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/case-sensitive-paths-webpack-plugin/builds/211017442
+# Created travis.yml  : No
+# Maintainer          : Devendranath Thadi <devendranath.thadi3@gmail.com>
+#
+# Script License      : MIT License
```

Comment:
Script License should always be "Apache 2.0"

Reply:
Done.